stdlib/email/src/main/java/org/ballerinalang/stdlib/email/util/SmtpUtil.java

```diff
@@ -125,4 +225,8 @@ private static String getNullCheckedString(String string) {
         return string == null ? "" : string;
     }
 
+    private static boolean stringIsNotNullOrEmpty(String string) {
```

Comment:
Shall we rename method name to something like `isNotEmpty`. it will cover the null check as well.

Reply:
Renamed.